### PR TITLE
Add article reference for integration Lighthouse CI

### PR DESCRIPTION
**Pull request by akanshgulati**
The article explains exactly a use-case of Lighthouse CI for static site generators which can run a local server for serving files and test performance.

### CODE REVIEW

**Comment by patrickhulce** [comment]
Thanks very much for the contribution @akanshgulati, great job on the article! 😃 

I'm not sure how I feel about linking to what basically amounts to alternative documentation in "Related Projects" on the main readme though. There are a few suggestions in the article that skip past some of the disclaimers we were very careful about in the official documentation, so I'm nervous to give it such prominence as-is.

Proposal: Maybe we can work together to clear up a few of the discrepancies with the official docs in the post and add a "Community Guides" section for this sort of thing? :)

**Comment by akanshgulati** [comment]
@patrickhulce I totally agree that it's not a project and should not be added under that section (missed by me while creating PR). I tried to explain a particular use-case where Lighthouse CI is being used for static website performance testing and as you said, this can be added under a "Community Guides" section instead.
Regarding the changes, I think it's best to keep the article in sync with Lighthouse CI team, please suggest changes. 
Article source file - https://github.com/akanshgulati/personal-blog-gatsby-markdown/blob/master/src/posts/integrating-lighthouse-ci/integrate-lighthouse-ci.md

**Comment by patrickhulce** [comment]
OK @akanshgulati I opened a PR with some tweaks over there :)

**Comment by akanshgulati** [comment]
> Thanks very much for the contribution @akanshgulati, great job on the article! 😃
> 
> I'm not sure how I feel about linking to what basically amounts to alternative documentation in "Related Projects" on the main readme though. There are a few suggestions in the article that skip past some of the disclaimers we were very careful about in the official documentation, so I'm nervous to give it such prominence as-is.
> 
> Proposal: Maybe we can work together to clear up a few of the discrepancies with the official docs in the post and add a "Community Guides" section for this sort of thing? :)

Time to add a new section 🥇

**Comment by akanshgulati** [comment]
@googlebot  I signed it!

File: README.md [hunk 1]
```diff
@@ -36,3 +36,5 @@ addons:
 - [Lighthouse CI GitHub Action](https://github.com/treosh/lighthouse-ci-action) - Automatically run Lighthouse CI on every PR with GitHub Actions, no infrastructure required.
 
 - [Learning Lighthouse CI](https://github.com/hchiam/learning-lighthouse-ci) - A minimal example repo that you can use as a template when starting from scratch, offers a beginner-friendly quickstart guide using create-react-app.
+
```
Comment (patrickhulce) [inline README.md]:
```suggestion

## Community Guides
A collection of unofficial blog posts, tutorials, and guides written by the community on using Lighthouse CI. 

**NOTE:** This is not official documentation. You're encouraged to familiarize yourself with Lighthouse CI and read through [Getting Started](./docs/getting-started.md) before continuing.
```

WDYT?

Comment (akanshgulati) [reply README.md]: Yeah, LGTM.

**Pull request merged**